action.yml

```diff
@@ -14,20 +14,6 @@ runs:
         echo "::error title=⛔ error hint::Support Linux Only"
         exit 1
 
-    - name: Remove unwanted stuff to free up disk image
-      shell: bash
-      run: |
-        sudo rm -rf /usr/share/dotnet
-        sudo rm -rf /usr/local/lib/android
-        sudo rm -rf /opt/ghc
-        sudo rm -rf "/usr/local/share/boost"
-        sudo rm -rf /opt/hostedtoolcache/CodeQL
- 
-        sudo docker image prune --all --force
- 
-        sudo swapoff -a
-        sudo rm -f /mnt/swapfile
-
```

Comment:
If your workload require more disk space then this might be still good to have?

Reply:
https://github.com/marketplace/actions/maximize-build-disk-space